webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DSRemoteTracingstoreClient.scala

```diff
@@ -0,0 +1,73 @@
+package com.scalableminds.webknossos.datastore.services
+
+import com.google.inject.Inject
+import com.scalableminds.util.tools.{Fox, FoxImplicits}
+import com.scalableminds.webknossos.datastore.DataStoreConfig
+import com.scalableminds.webknossos.datastore.dataformats.zarr.ZarrSegmentationLayer
+import com.scalableminds.webknossos.datastore.jzarr.{OmeNgffHeader, ZarrHeader}
+import com.scalableminds.webknossos.datastore.rpc.RPC
+import com.typesafe.scalalogging.LazyLogging
+import play.api.inject.ApplicationLifecycle
+import play.api.libs.json.JsObject
+
+class DSRemoteTracingstoreClient @Inject()(
+    rpc: RPC,
+    config: DataStoreConfig,
+    val lifecycle: ApplicationLifecycle,
+) extends LazyLogging
+    with FoxImplicits {
+
+  // TODO key check in tracing store
```

Comment:
I think it is fair to say that if the user has the accessToken, all the below requests can be considered authorized. We can therefore skip the dataStoreKey validation, which would be a bit more complicated here, since the tracingstore would have to forward that to webKnossos.

So please remove this comment as well as the dataStoreKey logic from this file.